### PR TITLE
"Add Veniamin Stakhovsky to the student list"

### PR DESCRIPTION
**Pull request by vstakhovsky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,3 +8,4 @@
 | № | Имя Фамилия     | Любимый фильм                          |
 | - |:---------------:| --------------------------------------:|
 | 0 | Ава Катушка     | Три идиота                             |
+| 4 | Веня Стаховский | Интерстеллар                           |
```
Comment (ava8katushka) [inline README.md]: How is it possible that after number 0 follows number 4?

**Review by ava8katushka — CHANGES_REQUESTED** [review]
Please address the comments and resolve the conflicts.

File: README.md (hunk 1 above)
Comment (ava8katushka) [reply README.md]: *number 4 goes after number 0. It shouldn't be. Please fix.

Comment (vstakhovsky) [reply README.md]: I used number №4 because there were 3 pull requests before me

Comment (ava8katushka) [reply README.md]: Well, it's not how git works. You should ignore all the other pull requests until they are accepted and merged. 
But you should pull changes that did already happen to your repository.

Comment (ava8katushka) [reply README.md]: You should also resolve the conflicts.

Comment (vstakhovsky) [reply README.md]: Ava, Sorry, i don't understand how to pull changes from the main repository to my own repository.

Comment (ava8katushka) [reply README.md]: You can start with resolving conflicts:

**Comment by vstakhovsky** [comment]
@ava8katushka done, take a look please

**Pull request merged**